services/py-api/src/server/routes/participants_routes.py

```diff
@@ -39,9 +40,9 @@ def _h_handler(
     "", status_code=201, responses={201: {"model": ParticipantRegisteredResponse}, 409: {"model": ErrResponse}}
 )
 async def create_participant(
-    response: Response, input_data: ParticipantRequestBody, handler: ParticipantHandlers = Depends(_p_handler)
+    response: Response, input_data: ParticipantRequestBody, jwt_token: Optional[str]=Query(None), handler: ParticipantHandlers = Depends(_p_handler)
```

Comment:
This works, however, I would prefer if we followed the fastapi documentation for defining optional query parameters
Here is a link to the docs: https://fastapi.tiangolo.com/tutorial/query-params/#optional-parameters
Here is an example from the docs: 
```Python
from typing import Union

from fastapi import FastAPI

app = FastAPI()


@app.get("/items/{item_id}")
async def read_item(item_id: str, q: Union[str, None] = None):
    if q:
        return {"item_id": item_id, "q": q}
    return {"item_id": item_id}
```

where q is the query parameter

Reply:
Fixed in commit a9a6b60a1c533cba63bd89fc5bee4362218c00f7